src/main/java/net/glowstone/generator/objects/OreVein.java

```diff
@@ -34,25 +40,25 @@ public boolean generate(World world, Random random, int sourceX, int sourceY, in
             double originY = dy1 + (dy2 - dy1) * i / amount;
             double originZ = dz1 + (dz2 - dz1) * i / amount;
             double q = random.nextDouble() * amount / 16.0D;
-            double hRadius = (Math.sin(i * (float) Math.PI / amount) + 1 * q + 1) / 2.0D;
-            double vRadius = (Math.sin(i * (float) Math.PI / amount) + 1 * q + 1) / 2.0D;
-            for (int x = (int) (originX - hRadius); x <= (int) (originX - hRadius); x++) {
-                double pX = (x + 0.5D - originX) / hRadius;
-                pX *= pX;
-                if (pX >= 1) {
+            double radiusH = (Math.sin(i * (float) Math.PI / amount) + 1 * q + 1) / 2.0D;
+            double radiusV = (Math.sin(i * (float) Math.PI / amount) + 1 * q + 1) / 2.0D;
+            for (int x = (int) (originX - radiusH); x <= (int) (originX - radiusH); x++) {
+                double px = (x + 0.5D - originX) / radiusH;
+                px *= px;
```

Comment:
I'd really like to decipher the name of this variable.

Reply:
These are transformed coordinates, with the coordinate space defined to turn the ore vein (which is a spheroid having one radius on the xz plane and another on the y axis) into the unit sphere. Hopefully my latest commit clears things up.